### DISCUSSION TITLE
BeginPopupModal instant crash

### DISCUSSION THREAD

**Discussion by Hrothgor** [ocornut/imgui#5067] [New Users Build/Link/Run issues ONLY!]
I am using a lot of ImGui features, everything work perfectly, but when I try to use Modal it always crash.

Just this simple code crash: 
```
        ImGui::Begin("test");
        if (ImGui::Button("Test"))
            ImGui::OpenPopup("Test?");
        if (ImGui::BeginPopupModal("Test?", NULL, ImGuiWindowFlags_AlwaysAutoResize))
        {
            ImGui::Text("Hello world!");
            if (ImGui::Button("OK", ImVec2(120, 0))) { ImGui::CloseCurrentPopup(); }
            ImGui::EndPopup();
        }
        ImGui::End();
```
It give me and error when I click the button to open the modal:
`const T& ImVector<T>::operator[](int) const [with T = short unsigned int]: Assertion `i >= 0 && i < Size' failed.`

Anyone has an idea why ?

**Comment by PathogenDavid** [comment]
I cannot repro this crash on v1.87 or the current master/docking branches.

It would be helpful if you [posted an issue and filled out the entire template](https://github.com/ocornut/imgui/issues/new) and provided a stack trace of the crash since that assert alone is pretty vague.

**Reply by Hrothgor** [reply]
Thanks for the answer, but I used raylib binding with ImGui, and the problem was from raylib. They updated it, now it works.

**Reply by PathogenDavid** [reply]
Glad to hear you got things sorted!
